tests/unit/test_create_async_query.py

```diff
@@ -31,3 +31,7 @@ def test_process_async_query_keeps_job_status_updated(self, mock_job_status):
     def test_create_async_query_calls_process_async_func_with_correct_args(self, mock_process):
         # handler(self.mock_event, 'context')
```

Comment:
delete?

Reply:
The change in this file was erroneusly included in this PR. I updated it to exclude this change. I will update the tests in this file in a separate PR.

Reply:
Filed the PR in #161.